lifecycle-lab/src/Component/Form.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+class Form extends React.Component{
+ constructor(props){
+     super(props)
+     this.state ={
+         text: ""
+     }
+     this.handleSubmit = this.handleSubmit.bind(this)
```

Comment:
You only need to do this if you are creating the methods on in es5 format ex:

`handleSubmit(event){`
`}`